scopes/dependencies/dependency-resolver/manifest/workspace-manifest-factory.ts

```diff
@@ -122,12 +125,14 @@ export class WorkspaceManifestFactory {
     {
       dependencyFilterFn,
       filterComponentsFromManifests,
+      filterExtensionDependencies,
       referenceLocalPackages,
       rootDependencies,
       rootPolicy,
     }: {
       dependencyFilterFn?: DepsFilterFn;
       filterComponentsFromManifests?: boolean;
+      filterExtensionDependencies?: boolean;
```

Comment:
It is not very clear what this means. I think it would be better to name it either `filterOutExtensionDependencies` or `excludeExtensionDependencies`.

Reply:
renamed